Assets/Scripts/Map/RobotSpawners/PatrollingRobotSpawner.cs

```diff
@@ -18,9 +20,9 @@ public void SetPatrolling(PatrollingMap map, PatrollingTracker tracker)
         }
 
         protected override MonaRobot CreateRobot(float x, float y, float relativeSize, int robotId, IPatrollingAlgorithm algorithm,
-            SimulationMap<Tile> collisionMap, int seed)
+            SimulationMap<Tile> collisionMap, int seed, Color32 color)
         {
-            var robot = base.CreateRobot(x, y, relativeSize, robotId, algorithm, collisionMap, seed);
+            var robot = base.CreateRobot(x, y, relativeSize, robotId, algorithm, collisionMap, seed, color);
```

Comment:
The robot color is not that useful anymore. If we need it it can just be done on the algorithm side.

Reply:
I think that we should keep this, because then we can also outline the robot with the assigned color, and we also give them a unique color :)

Reply:
ok